cdap-unit-test/src/test/java/co/cask/cdap/spark/SparkTestRun.java

```diff
@@ -294,6 +302,50 @@ public void testScalaSparkWithObjectStore() throws Exception {
   }
 
   @Test
+  public void testScalaSparkCrossNSStream() throws Exception {
```

Comment:
Yes we have one here: https://github.com/caskdata/cdap/pull/6036/files#diff-9eaa30d248fe113b4582df9765291177R59

This one is for both dataset&stream.